### DISCUSSION TITLE
Keep the "State" option blank by default in the WooCommerce Checkout page

### DISCUSSION THREAD

**Discussion by srottam** [woocommerce/woocommerce#49736] [Checkout Flow]
Right now, the "State" option is never blank, which results in improper address capture and hence differences in shipping rates if the end user is not careful. Have the State option populated by the pin code entered or leave it blanck (but required) until the customer selected it from the list. Either of the two is necessary.

**Comment by senadir** [comment]
[This PR should handle making it empty by default](https://github.com/woocommerce/woocommerce/pull/49616).
